files/graphite-scripts/letsencrypt-metrics

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/php
+<?php
+
+if (!file_exists('/opt/graphite/graphite_functions.php') || !file_exists('/etc/nginx/letsencrypt')) {
+    exit(0);
+}
+
+require_once '/opt/graphite/graphite_functions.php';
+
+list($serverName,) = explode(".", trim(`hostname -f`));
+
+$now = time();
+$numberSANS = 0;
+
+foreach (glob("/etc/nginx/letsencrypt/*crt") as $certFile) {

```

Comment:
was going to ask if this path could be changed, i.e. so we could change the path if we decided to store the certs elsewhere for SSP... but it doesn't look like there's any way to set the parameters for individual metrics collector scripts in this module

Reply:
or maybe there is, we could make this a puppet template and parameterise the cert path?

Reply:
The naming of metrics etc make it relevant to our Let's Encrypt tools.

The path is hard coded currently, to `/etc/nginx/letsencrypt/` in the puppet install.